src/core/public/public.api.md

```diff
@@ -18,57 +17,59 @@ import { UserProvidedValues as UserProvidedValues_2 } from 'src/core/server/type
 
 // @public
 export interface App extends AppBase {
-    appRoute?: string;
-    chromeless?: boolean;
-    mount: AppMount | AppMountDeprecated;
+  appRoute?: string;
+  chromeless?: boolean;
+  mount: AppMount | AppMountDeprecated;
 }
 
 // @public (undocumented)
 export interface AppBase {
-    capabilities?: Partial<Capabilities>;
-    chromeless?: boolean;
-    euiIconType?: string;
-    icon?: string;
-    id: string;
-    // @internal
-    legacy?: boolean;
-    navLinkStatus?: AppNavLinkStatus;
-    order?: number;
-    status?: AppStatus;
-    title: string;
-    tooltip?: string;
-    updater$?: Observable<AppUpdater>;
+  capabilities?: Partial<Capabilities>;
+  // Warning: (ae-forgotten-export) The symbol "AppCategory" needs to be exported by the entry point index.d.ts
```

Comment:
@pgayvallet do you know how to resolve this? I'm not sure which `index.d.ts` this is referring to...

Reply:
You should import and re-export `AppCategory` from `src/core/public/index.ts`. This would also help client-side code to import it directly from `core/public`.